iOS/Views/MapViewCards/RouteResultCard.cs

```diff
@@ -0,0 +1,164 @@
+                headerLabel.TrailingAnchor.ConstraintEqualTo(closeButton.LeadingAnchor, -ApplicationTheme.Margin),
+                //clear button
+                closeButton.TrailingAnchor.ConstraintEqualTo(TrailingAnchor, -ApplicationTheme.Margin),
+                closeButton.CenterYAnchor.ConstraintEqualTo(headerLabel.CenterYAnchor),
+                closeButton.WidthAnchor.ConstraintEqualTo(32),
+                closeButton.HeightAnchor.ConstraintEqualTo(32),
+                // stops view
+                _stopsTable.LeadingAnchor.ConstraintEqualTo(LeadingAnchor),
+                _stopsTable.TopAnchor.ConstraintEqualTo(_routeDurationLabel.BottomAnchor, ApplicationTheme.Margin),
+                _stopsTable.TrailingAnchor.ConstraintEqualTo(TrailingAnchor, -ApplicationTheme.Margin),
+                // image
+                travelModeImageView.LeadingAnchor.ConstraintEqualTo(LeadingAnchor, ApplicationTheme.Margin),
+                travelModeImageView.TopAnchor.ConstraintEqualTo(_routeDurationLabel.TopAnchor),
+                travelModeImageView.BottomAnchor.ConstraintEqualTo(_routeDurationLabel.BottomAnchor),
+                travelModeImageView.WidthAnchor.ConstraintEqualTo(32),
+                // walk time label
+                _routeDurationLabel.TopAnchor.ConstraintEqualTo(headerLabel.BottomAnchor, ApplicationTheme.Margin),
+                _routeDurationLabel.LeadingAnchor.ConstraintEqualTo(travelModeImageView.TrailingAnchor, ApplicationTheme.Margin),
+                // update bottom size
+                BottomAnchor.ConstraintEqualTo(_stopsTable.BottomAnchor, ApplicationTheme.Margin)
+            });
+
+            closeButton.TouchUpInside += Close_Clicked;
+
+            _viewModel.PropertyChanged += ViewModel_PropertyChanged;
+        }
+
+        /// <summary>
+        /// Update UI for viewmodel property changes
+        /// </summary>
+        /// <param name="sender"></param>
+        /// <param name="e"></param>
+        private void ViewModel_PropertyChanged(object sender, System.ComponentModel.PropertyChangedEventArgs e)
+        {
+            if (e.PropertyName != nameof(_viewModel.CurrentRoute))
+            {
+                return;
+            }
+
+            Route firstRoute = _viewModel.CurrentRoute?.Routes?.FirstOrDefault();
+
+            if (firstRoute == null)
+            {
+                _stopsTable.Source = null;
+                return;
+            }
+
+            // Build up the walk time label
+            StringBuilder walkTimeStringBuilder = new StringBuilder();
+
+            // Add walk time and distance label
+            if (firstRoute.TotalTime.Hours > 0)
+            {
+                walkTimeStringBuilder.Append($"{firstRoute.TotalTime.Hours} h {firstRoute.TotalTime.Minutes} m");
+            }
+            else
+            {
+                walkTimeStringBuilder.Append($"{firstRoute.TotalTime.Minutes + 1} min");
```

Comment:
Is there a convenient way to localize "min" here? Is that localization handled elsewhere?